README.md

```diff
@@ -226,6 +226,41 @@ Follow Helm Chart installation instructions [here](https://docs.openshift.com/co
 
 ## Additional information
 
+### Additional Workflow Namespaces
+
+If you want to deploy workflows on other namespaces, make sure you follow these steps:
+
+ - For the cluster Sonataflow services to accept the traffic from the workflows, set this label:
+   ```console
+      oc label ns my-additional-workflow-ns rhdh.redhat.com/workflow-namespace=""
+   ```
+
+ - Set a network policy to allow only RHDH and Sonatflow services traffic. Taken from the `charts/orchestrator/templates/network-policy.yaml`
+
+   ```console
+      oc create -f <<EOF
+      apiVersion: networking.k8s.io/v1
+      kind: NetworkPolicy
+      metadata:
+        name: allow-rhdh-to-sonataflow-and-workflows
+        # Sonataflow and Workflows are using the same namespace.
```

Comment:
` are using the same namespace` the goal is to use different namespace, can you clarify?

Reply:
you are correct, this comment is uneeded